pages/health-care/apply/application.md

```diff
@@ -1,22 +0,0 @@
----
-title: Apply for Health Care
-entryname: hca
-layout: page-react.html
-display_title: Apply Now
-description: Apply for VA health care benefits. Find out which documents you’ll need, and start your online application today.
-body_class: page-healthcare
-in_maintenance: false
-maintenance_line1: We’re sorry. The health care application is currently down while we fix a few things. We’ll be back up as soon as we can.
-maintenance_line2: In the meantime, you can call 1-877-222-VETS (<a href="tel:+18772228387">1-877-222-8387</a>), Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (<abbr title="eastern time">ET</abbr>) and press 2 to complete this application over the phone.
-collection: healthCare
-spoke: Get benefits
-order: 4
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/health-care/">Health Care</a></li>
-    <li><a aria-current="page" href="/health-care/how-to-apply/">Apply for VA Health Care</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-3f08f8ea75c3be0849d472b7d63ef2fcR5. Breadcrumbs look good.